src/util/jira.ts

```diff
@@ -1,13 +1,14 @@
 import { JiraAuthor } from "../interfaces/jira";
+import _ from "lodash";
 
 export const getJiraAuthor = (...authors: (Author | undefined)[]): JiraAuthor => {
 	const author = Object.assign({}, ...authors);
-	return author.login || author.name ? {
-		avatar: author.avatar_url || author.avatarUrl || `https://github.com/users/${author.login}.png`,
+	return author.login || author.name ? _.pickBy({
+		avatar: author.avatar_url || author.avatarUrl || (author.login ? `https://github.com/users/${author.login}.png` : undefined),
 		name: author.name || author.user?.name || author.login || author.email?.match(/^(.*)@/)?.pop() || "unknown",
 		email: author.email || `${author.login}@noreply.user.github.com`,
-		url: author.html_url || author.url || author.user?.url || `https://github.com/users/${author.login}`
-	} : {
+		url: author.html_url || author.url || author.user?.url || (author.login ? `https://github.com/users/${author.login}` : undefined)
+	}) as JiraAuthor : {
```

Comment:
That's definirely should be done in another PR. We already faced problems in the past when doing refactoring with lodash

Reply:
this was breaking some tests and it did find some edge cases, hence the fix.